### PR TITLE
HPCC-16096 Add "legacy" mode for results.

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-16096

Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/ResultsWidget.js [hunk 1]
```diff
@@ -180,6 +193,19 @@ define([
                         }
                     }
                 });
+            } else if (params && params.legacyMode) {
+                return new DelayLoadWidget({
+                    id: id,
+                    title: "[L] " + row.Name,
```
Comment (miguelvazq) [inline esp/src/eclwatch/ResultsWidget.js]: is "[L]" going to be sufficient for users to understand its meaning?

Comment (GordonSmith) [inline esp/src/eclwatch/ResultsWidget.js]: We use it in several places for the "L"egacy tabs (plus they will very quickly associate it with the button they just pressed)

Comment (miguelvazq) [inline esp/src/eclwatch/ResultsWidget.js]: Minor comment otherwise, ok.

**Pull request merged**